circle/functions

```diff
@@ -128,6 +129,39 @@ install_google_cloud_sdk() {
   fi
 }
 
+install_azure_cli() {
+  if ! which az >/dev/null ; then
+    info "Modifying source lists..."
+    AZ_REPO=$(lsb_release -cs)
+    echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $AZ_REPO main" | \
+    tee /etc/apt/sources.list.d/azure-cli.list
+    if ! which az >/dev/null ; then
+      sudo apt-get install -y curl
+    fi
+
+    info "Getting the Microsoft signing key.."
+    if ! curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add - ; then
+      error "Could not obtain Microsoft signing key.."
+      return 1
+    fi
+    apt-key adv --keyserver packages.microsoft.com --recv-keys 52E16F86FEE04B979B07E28DB02C46DF417A0893
+
+    info "Installing azure cli dependencies..."
+    apt-get update
```

Comment:
I don't think this is going to work. I think the circle machines are alpine based and do not have apt-get. The last thing I install what `yq` and need to use `apk`